PSRabbitMq/Connect-RabbitMqChannel.ps1

```diff
@@ -84,7 +92,12 @@
     {
         $Channel = $Connection.CreateModel()
 
-         Write-Progress -id 10 -Activity 'Create SCMB Connection' -Status 'Attempting connection to channel' -PercentComplete 80
+        Write-Progress -id 10 -Activity 'Create SCMB Connection' -Status 'Attempting connection to channel' -PercentComplete 80
+
+        #Actively declare the Exchange (as non-autodelete, non-durable)
+        if($ExchangeType -and [string]::Empty -ne $Exchange) {
```

Comment:
Do we need to test for $Exchange -ne $null as well here?  Should we be using:

-not [String]::IsNullOrEmpty($Exchange)